client/react/components/Search/styles.css

```diff
@@ -0,0 +1,15 @@
+.search {
+    text-align: center;
```

Comment:
Order of css is typically:

- display
- width
- flexbox stuff
- margins
- padding
- font stuff
- line stuff
- wrapping stuff
- z-index at the end